src/org/labkey/test/tests/SampleTypeParentColumnTest.java

```diff
@@ -301,9 +302,24 @@ public void testValidAliasNames()
         SampleTypeAPIHelper.createEmptySampleType(PROJECT_NAME + "/" + SUB_FOLDER_NAME, definition);
         projectMenu().navigateToFolder(PROJECT_NAME, SUB_FOLDER_NAME);
 
-        log("Import samples that have a parent alias column.");
+        log("Verify parent alias display in sample type properties and download template.");
         SampleTypeHelper sampleHelper = new SampleTypeHelper(this);
-        sampleHelper.goToSampleType(SAMPLE_TYPE_NAME).bulkImport(sampleText);
+        sampleHelper.goToSampleType(SAMPLE_TYPE_NAME);
+        Assert.assertEquals("Unexpected parent import alias display", PARENT_COLUMN_1, sampleHelper.getDetailsFieldValue("Parent Import Alias(es)"));
+        ImportDataPage importDataPage = sampleHelper.getSamplesDataRegionTable().clickImportBulkData();
+        try
+        {
+            List<String> actualHeaderValues = importDataPage.getTemplateColumnHeaders();
+            Assert.assertTrue("Download template file does not contain import alias", actualHeaderValues.contains(PARENT_COLUMN_1));
+        }
+        catch (IOException rethrow)
+        {
+            throw new RuntimeException(rethrow);
+        }
```

Comment:
More or less just a style comment, but I usually don't catch and rethrow exceptions in the test case I just add a throws IOException to the to the function declaration:
`public void testValidAliasNames() throws IOException`

Reply:
that works for me. changed.